### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by jgiwinski**
## Is this a fix or a feature?
Feature 

## What is the change?
- Add and style login form
- Add and style create account form
- Add functionality to buttons

## Where should the reviewer start?
- LoginForm.vue 

## How should this be tested?
-NA

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
```
Comment (mattumland) [inline public/index.html:2]: Nice, good for accessibility

File: src/App.vue [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
+  data() {
+    return{
+      loggedIn: false
+    }
+  },
+  methods: {
+    updateLogin(event) {
+      event.preventDefault();
+      !this.loggedIn ? this.loggedIn = true : this.loggedIn = false 
+      console.log(event)
```
Comment (mattumland) [inline src/App.vue:31]: We'll want to remove these later.

File: src/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+$black: #0D0D0D;
+$darkestRed: #260609;
+$mediumRed: #591419;
+$lightRed: #8C232C;
+$gray: #D9D9D9;
```
Comment (Jacksonmcguire) [inline src/_variables.scss:5]: Are we positive this is how grey is spelled in america?

File: src/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+$black: #0D0D0D;
+$darkestRed: #260609;
```
Comment (mattumland) [inline src/_variables.scss:2]: DARKEST RED 😈

File: src/components/Header.vue [hunk 5]
```diff
@@ -29,10 +32,10 @@ export default {
         width: 100%;
         top: 0;
         vertical-align:bottom;
```
Comment (mattumland) [inline src/components/Header.vue:34]: might want to put a space after the :

File: src/components/LoginForm.vue [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  .form-view {
+    background-image: url('../assets/red_seats.jpg');
+    height: 90%;
+    padding-top: 90px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: 50% 70%;
+  }
+
+  .login-form  {
```
Comment (mattumland) [inline src/components/LoginForm.vue:122]: I wonder if any of this can be made into a mixin

**Review by mattumland — APPROVED** [review]

**Pull request merged**